lib/travis/api/build/app.rb

```diff
@@ -20,16 +20,17 @@ class App < Sinatra::Base
         end
 
         configure do
-          use Sentry unless Travis::Build.config.sentry_dsn.empty?
-          use Metriks unless Travis::Build.config.librato.email.empty? ||
-                             Travis::Build.config.librato.token.empty? ||
-                             Travis::Build.config.librato.source.empty?
+          use Sentry unless Travis::Build.config.sentry_dsn.nil?
+          use Metriks unless Travis::Build.config.librato.email.nil? ||
+                             Travis::Build.config.librato.token.nil? ||
+                             Travis::Build.config.librato.source.nil?
```

Comment:
Glad you asked about this, because maybe you have a solution. Really neither nil or empty are acceptable. This line of code never gets used in .com/.org but in Enterprise these get passed as nils when not set. Which results in an error of `empty?` not being available on `nil`. The original code before getting converted to `travis-config` checked if `ENV['TRAVIS_LIBRATO_EMAIL']` was true. Which would cover empty strings but not nils. 

What I really want is `.blank?` but that doesn't seem to be available here (also I'm not sure we set librato source ever in Enterprise. What exactly is that one, that seems to be new).